src/spuco/robust_train/cnc_train.py

```diff
@@ -180,6 +180,7 @@ def __init__(
         self.sampler = sampler
         self.verbose = verbose
         self.device = device
+        self.name = 'CnC'
```

Comment:
what is this for?

Reply:
the current base class requires self.name